### PR TITLE
Add individual dat init error states. Closes #284

### LINKED ISSUE #284: Bug: Moving folder after download causes crash on restart
A user moved their dataset out of `Downloads/dat` and then on restart got this:

```
(dmkanter) the cli seems to be working fine and sharing.  now however when I load the electron client it exits due to a crash, and asks me if I want to clear the db and exit, or delete all data and exit. I'm on version 1.0.2 of electron client, not latest (because for some reason couldn't execute installer for 1.0.3)
```

### PR DESCRIPTION
**Pull request by juliangruber**
Closes #284

This needs UI still, it currently looks like this:

screen shot 2017-03-17 at 15 08 46

### CODE REVIEW

File: elements/table.js [hunk 1]
```diff
@@ -137,6 +137,25 @@ function tableElement (dats, send) {
 }
 
 function row (dat, send) {
+  if (dat instanceof Error) return html`
+    <tr>
+      <td class="cell-1">
+        <div class="w2 center">
+          OOPS
```
Comment (juliangruber) [inline elements/table.js]: could render the individual error message here to via `dat.message`

**Comment by Kriesse** [comment]
@juliangruber Cool! How can I reproduce this locally to style it?

**Comment by juliangruber** [comment]
@Kriesse download a dat from the network (don't need to finish it), then close the app, remove `~/Downloads/dat*`, then open again.

**Comment by karissa** [comment]
yes!

**Comment by Kriesse** [comment]
@juliangruber That didn't quite work, I get this –
- I can only close the app
- The modal style seems to be broken
Any ideas?
image

**Comment by karissa** [comment]
Can this be merged?

**Comment by juliangruber** [comment]
i'll rebase this onto `master` first, then @Kriesse can take on styling this :)

**Comment by juliangruber** [comment]
also this is still blocked by https://github.com/datproject/multidat/pull/7

**Comment by juliangruber** [comment]
@Kriesse can you give this another spin now?

**Comment by Kriesse** [comment]
@yoshuawuyts Can you help to resolve the conflicts? I believe other than that this is ok to merge.

**Comment by juliangruber** [comment]
@Kriesse do you think this is cool styling wise?

**Comment by Kriesse** [comment]
@juliangruber Yes it looks good now locally – but apparently my merge commit broke stuff. Not sure what's going on.

It looks like this after i reverted the merge commit:
image

**Comment by Kriesse** [comment]
@juliangruber Ok to merge now, thanks for the help 👍

**Comment by juliangruber** [comment]
❤️ thank you everyone!

**Pull request merged**